src/components/filesystem/index.js

```diff
@@ -317,21 +316,21 @@ module.exports = {
 		this.showHexViewer = true;
 	},
 
-	updateCurrentDir: function() {
+	updateCurrentDir: function(refreshSharing) {
```

Comment:
I'm a little nervous about coupling the sharing update to the dir directly like this. There are also other calls to this method that are not passing an arg now.

Reply:
It is only relevant in the one case when forceSharedWithUpdate is triggered in order to refresh the sharedWithData data structure.